lms/lib/comment_client/user.py

```diff
@@ -206,6 +206,10 @@ def get_course_social_stats(course_id, end_date=None):
     if end_date:
         params.update({'end_date': end_date.isoformat()})
 
+    # Pass in user ids if available
+    if user_ids:
+        params.update({'user_ids[]': user_ids})
```

Comment:
Hmm, I'm not sure we want to send this list of users as part of the API request. I'm afraid the list could become very long and exceed the HTTP request size limit on our servers.